hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/views/heatMap/heatmap.less

```
@@ -0,0 +1,60 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+.heatmap-container {
+
+  .go-back-button {
+    display: inline-block;
+    margin-left: 20px;
+  }
+
+  .entity-dropdown-button {
+    display: inline-block;
+    margin-left: 30px;
+  }
+
+  .date-dropdown-button {
+    display: inline-block;
+    margin-left: 45px;
+  }
+
+  .input-bar {
+    display: inline-block;
+    margin-left: 25px;
+  }
+
+  .input {
+    padding-left: 5px;
+    margin-right: 10px;
+    display: inline-block;
+    width: 350px;
+  }
+
+  .heatmapinformation {
+    font-size: 14px;
+    fill: rgba(0, 0, 0, 0.65) !important;
+    line-height: 1.5;
+    font-feature-settings: 'tnum', "tnum";
+    height: 800;
+  }
+
+}
+
+.ant-drawer-open .ant-drawer-mask {
+  opacity: 0 !important;
+}
```

add new line at the end
```suggestion
}

```